recipes/oslo.i18n/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{%set name = "oslo.i18n" %}
+{%set version = "3.7.0" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "f534a69013891b64320f5a9d444a4ae0e28d388b6b97f63ee5c7532292454287" %}
+{%set build_num = "0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.8
+
+  run:
+    - python
+    - babel >=2.3.4
+    - six >=1.9.0
+    - fixtures
+
+test:
+  imports:
+    - oslo_i18n
+    - oslo_i18n.log
+    - oslo_i18n.fixture
+
+about:
+  home: http://docs.openstack.org/developer/oslo.i18n/
+  license: Apache 2.0
+  # license_file: No MANIFEST.in included in distribution
+  license_family: Apache
+  summary: 'Utilities for working with internationalization (i18n) features, especially translation for text strings in an application or library.'
+  doc_url: http://docs.openstack.org/developer/oslo.i18n/
+  dev_url: http://git.openstack.org/cgit/openstack/oslo.i18n
+
+extra:
+  recipe-maintainers:
+    - pmlandwehr
```

Comment:
Maybe add @anguslees based on his expressed interest in PR ( https://github.com/conda-forge/staged-recipes/pull/537 ).